UnoCheck/Checkups/XCodeCheckup.cs

```diff
@@ -35,7 +35,7 @@ public string VersionName
 
 		public override string Id => "xcode";
 
-		public override string Title => $"XCode {VersionName}";
+		public override string Title => $"Minimum XCode version ({VersionName})";
```

Comment:
It's not the _minimum_ as that would say 16.4 is fine when 16.3 is the minimum.

Maybe `Exact Xcode required version` ?

Reply:
@spouliot  It's a bit more complicated than that I guess.. There are cases when you can use either 16.3 or 16.4 because they do not bring any relevant change and we (dotnet) support both.

Reply:
No 😄 There's always a single unique version of Xcode **supported** by a .NET SDK version.

Others, earlier or future, versions _might_ work, fully or partially, but they have never been tested by Microsoft, nor will they ever do a service release for a newer Xcode version (you'll have to wait the next SDK release).

Reply:
@spouliot  what's the wording you suggest we use here? Based on your experience, what would be the most accurate?

Reply:
updated